x-pack/plugins/lens/public/embeddable/expression_wrapper.tsx

```diff
@@ -93,18 +95,9 @@ export function ExpressionWrapper({
           syncCursor={syncCursor}
           executionContext={executionContext}
           renderError={(errorMessage, error) => {
-            const messages = getOriginalRequestErrorMessages(error);
-            addUserMessages(
-              messages.map((message) => ({
-                uniqueId: message,
-                severity: 'error',
-                displayLocations: [{ id: 'visualizationOnEmbeddable' }],
-                longMessage: message,
-                shortMessage: message,
-                fixableInEditor: false,
-              }))
-            );
-            onRuntimeError(messages[0] ?? errorMessage);
+            const messages = getOriginalRequestErrorMessages(error || null, docLinks);
+            addUserMessages(messages);
```

Comment:
Is it ok to replace this
```
 addUserMessages(
              messages.map((message) => ({
                uniqueId: message,
                severity: 'error',
                displayLocations: [{ id: 'visualizationOnEmbeddable' }],
                longMessage: message,
                shortMessage: message,
                fixableInEditor: false,
              }))
            );
```

with this? `addUserMessages(messages);`

Reply:
I moved the mapping inside the `getOriginalRequestErrorMessages` directly so we have to deal with `UserMessage`s only. From my tests I could not find any drawback of it, but open to discuss.

Reply:
No I am fine with that, I just want to be sure that we are not introducing any bug. I can't find anything and this seems good to me but @drewdaemon knows this service better. Drew do you want to take a look?

Reply:
Works for me 👍